### ISSUE TITLE
Add Debian dependency: touchegg

### ISSUE THREAD

**Issue by m-roberts** [JoseExposito/touche#7]
At least a `recommends` field for it would make sense, even if not a hard dependency. This would mean that if touchegg is not installed and touche is installed from a Debian repository, then touchegg will be installed also. I think that this is the ideal behaviour.

**Comment by JoseExposito** [comment]
Yes, this could be really convenient. However, I need to do some testing, because the package available in the Ubuntu repos (https://packages.ubuntu.com/groovy/touchegg) is really outdated and I don't want to pull that one by mistake.
Ideally that package should be updated, but it doesn't seem like there is any interest on it.

**Comment by archisman-panigrahi** [comment]
> I don't want to pull that one by mistake.

This should not be a problem is touche is made available on the same ppa as touchegg. It would pull the ppa package, not the outdated package on the official repo
